common/src/main/java/com/wynntils/screens/maps/PoiManagementScreen.java

```diff
@@ -0,0 +1,243 @@
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Managers;
+import com.wynntils.features.map.MapFeature;
+import com.wynntils.models.map.pois.CustomPoi;
+import com.wynntils.screens.base.WynntilsScreen;
+import com.wynntils.screens.maps.widgets.PoiManagerWidget;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import java.util.ArrayList;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.components.AbstractWidget;
+import net.minecraft.client.gui.components.Button;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.client.resources.language.I18n;
+import net.minecraft.network.chat.Component;
+
+public class PoiManagementScreen extends WynntilsScreen {
+    private final MainMapScreen oldMapScreen;
+    private Button nextButton;
+    private Button previousButton;
+    private Button deleteAllButton;
+    private int deleteCount = 3;
+    private List<CustomPoi> waypoints;
+    private int pageHeight;
+    private int page;
+    private int spacingMultiplier = 20;
+    private final List<AbstractWidget> poiManagerWidgets = new ArrayList<>();
+
+    private PoiManagementScreen(MainMapScreen oldMapScreen) {
+        super(Component.literal("Poi Management Screen"));
+
+        this.oldMapScreen = oldMapScreen;
+    }
+
+    public static Screen create(MainMapScreen oldMapScreen) {
+        return new PoiManagementScreen(oldMapScreen);
+    }
+
+    @Override
+    public void onClose() {
+        McUtils.mc().setScreen(oldMapScreen);
+    }
+
+    @Override
+    protected void doInit() {
+        pageHeight = (this.height - 100) / spacingMultiplier;
+
+        this.addRenderableWidget(
+                new Button.Builder(Component.literal("X").withStyle(ChatFormatting.RED), (button) -> this.onClose())
+                        .pos(this.width - 40, 20)
+                        .size(20, 20)
+                        .build());
+
+        this.addRenderableWidget(
+                nextButton = new Button.Builder(Component.literal(">"), (button) -> {
+                            nextPage();
+                        })
+                        .pos(this.width / 2 + 2, this.height - 45)
+                        .size(20, 20)
+                        .build());
+
+        this.addRenderableWidget(
+                previousButton = new Button.Builder(Component.literal("<"), (button) -> {
+                            previousPage();
+                        })
+                        .pos(this.width / 2 - 22, this.height - 45)
+                        .size(20, 20)
+                        .build());
+
+        this.addRenderableWidget(
```

Comment:
This sounds very dangerous and drastic. Are we sure this is even needed? If someone mess up their waypoints that much, it is probably better to edit the json config files.

Reply:
Just because it was that way in legacy does not mean it was a good feature. :-) Apart from the risk of e.g. an autoclicker getting astray, there is also the risk that we somehow ourselves call such a function. I have a hard time seeing how this could ever be useful, compared to the risk that something goes wrong. 

Let's hear @kristofbolyai's opinion, but otherwise I'd recommend that you remove it, yes.

Reply:
You can reset pois via a command, if you really want to. So I don't think we need this.